projects/storefrontlib/shared/components/chat-messaging/avatar/avatar.component.html

```diff
@@ -0,0 +1,12 @@
+<!-- TODO: Verify current user to display left aligned messages -->
+<div
+  *ngIf="event"
+  class="cx-avatar"
+  [ngClass]="{ 'right-align': event.addedByAgent }"
```

Comment:
Let's give the power to the hosting app to set its alignment value
```suggestion
  [ngClass]="{ 'right-align': message.rightAlign }"
```

Reply:
yes, discussed something similar with Brian as well.
We can do this, but if the messageEvent is standard at SAP level, and the messageEvent would be similar in other cases as well.
We can keep the check for addedByAgent as well, which makes the rightAlign optional to pass.
What do you think @khristian23 ?

Reply:
Let me ask the Quotes team how they receive their entities in the UI.  If they also have the concept of an Event, then it will be ok.  Otherwise, we need to find a more generic naming

Reply:
Alright